Chinese/Casey/Casey.md

```diff
@@ -1,3 +1,5 @@
 Casey现在为udacity工作，自动合并GitHub的pull request。
 Casey is a help bot.
-Casey won't accept a pull request that a fix typo.
\ No newline at end of file
+Casey won't accept a pull request that a fix typo.
+直到有一天Casey收到来自转行开始学编程的YouTuber的pull request.
+他就是[CaseyNeistat](https://github.com/crossairplane/create-your-own-adventure/blob/neistat/Chinese/Casey/neistat/neistat.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.